ideascube/mediacenter/forms.py

```diff
@@ -17,6 +18,9 @@ class Meta:
         widgets = {
             'lang': LangSelect,
         }
+        labels = {
+            'original': _('Source file (maximum size : 500 Mo)'),
```

Comment:
Right, but this is kind of the django way: forms are generated from the model attributes. (it's horrendous if you ask me, but it's idiomatic)

Models even have the [`help_text`](https://docs.djangoproject.com/en/1.10/ref/models/fields/#help-text) attribute for forms! 🙄 

I don't really see any reason not to modify the model's `verbose_name` here. Especially since it already has a `verbose_name`, which means translators would now have to translate both "original" (the `verbose_name`) and "source file" (the `label`)